package.json

```diff
@@ -2,8 +2,12 @@
 	"name": "apache-camel-extension-pack",
 	"displayName": "Apache Camel Extension Pack",
```

Comment:
I think we are not allowed to use Apache Camel as a name here because of legal reasons. @davsclaus might prove me wrong.

Reply:
I created a dedicated issue https://github.com/camel-tooling/vscode-camel-extension-pack/issues/9 ad this PR is not touching the display name